packages/elements/src/pagination/index.ts

```diff
@@ -712,14 +560,16 @@ export class Pagination extends BasicElement {
           id="input"
```

Comment:
I think `id` is no longer required

Reply:
The `label` element has attribute `for` which using this `id`